contrib/endpoints/src/api_manager/method_impl.cc

```diff
@@ -51,17 +50,17 @@ void MethodInfoImpl::addAudiencesForIssuer(const string &issuer,
   if (iss.empty()) {
     return;
   }
+
   set<string> &audiences = issuer_audiences_map_[iss];
-  stringstream ss(audiences_list);
-  string audience;
-  // Audience list is comma-delimited.
-  while (getline(ss, audience, ',')) {
-    if (!audience.empty()) {  // Only adds non-empty audience.
-      std::string aud = utils::GetUrlContent(audience);
-      if (!aud.empty()) {
-        audiences.insert(aud);
-      }
+
+  // Audience list is comma-delimited with possible white spaces.
+  char *tokens = strtok(const_cast<char *>(audiences_list.c_str()), " ,");
```

Comment:
Actually, there is code in fetch_metadata.cc that is already doing this:

 esp_get_service_account_auth_token(
                          const_cast<char *>(body.data()), body.length(), ...)

The code passes this as input to grpc parser which only accepts a char * and NOT a const char *. Do we also need to fix this code?